source/documentation/tools.md

```diff
@@ -364,6 +364,8 @@ A [Python virtual environment](https://docs.python.org/3/tutorial/venv.html) (or
 
 ### Basic usage
 
+**NOTE:** You may need to delete the `.bash_aliases` file (`rm .bash_aliases`) from your home directory for pip to work properly within a virtual environment.
```

Comment:
I wonder if an explanation is helpful:
```suggestion
**NOTE:** To use pip with a virtual environment you should delete your `.bash_aliases` file from your home directory, if you have it: `rm .bash_aliases`.
Otherwise you'll find it always install packages to `~/.local` instead of the activated virtual environment. Only users who joined the Analytical Platform before 31st July 2020 has the problematic alias in contained in this file.
```